kvsqlite/sqlite.py

```diff
@@ -53,9 +53,14 @@ def __init__(
 
         self.is_running = True
 
-        self.__table_statement = 'CREATE TABLE IF NOT EXISTS "{}" (k VARCHAR(4096) PRIMARY KEY, v BLOB, expire_time INTEGER DEFAULT NULL)'.format(
+        self.__table_statement = 'CREATE TABLE IF NOT EXISTS "{}" (k VARCHAR(4096) PRIMARY KEY, v BLOB, expire_time INTEGER DEFAULT NULL) WITHOUT ROWID'.format(
             self.table_name
         )
+        self.__index_statement = (
+            'CREATE INDEX IF NOT EXISTS idx_expire_time ON "{}" (expire_time)'.format(
```

Comment:
It's better to use composite index for faster lookups:
```suggestion
            'CREATE INDEX IF NOT EXISTS idx_lookup ON "{}" (k, expire_time)'.format(
```